src/main/java/com/example/moviemanager/controller/dto/UpdateMovieRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.example.moviemanager.controller.dto;
+
+import com.example.moviemanager.repository.model.Movie;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public record UpdateMovieRequest(
+
+        @NotBlank(message = "Title is mandatory.")
```

Comment:
I think those validations can also be covered with parameterized test because now only create movie flow is covered